builder/vsphere/common/step_connect.go

```diff
@@ -71,9 +72,9 @@ func (s *StepConnect) Cleanup(state multistep.StateBag) {
 	driver := state.Get("driver").(driver.Driver)
 	errorRestClient, errorSoapClient := driver.Cleanup()
 	if errorRestClient != nil {
-		ui.Message("Error closing rest client session: " + errorRestClient.Error())
+		log.Printf("[WARN] Couldn't close rest client session. Could be already closed: %s", errorRestClient.Error())
```

Comment:
This is not really an issue because vSphere will automatically close if for whatever reason this failed for some other issue. Is that correct?

Reply:
Perhaps better wording and correction terms.

```suggestion
		log.Printf("[WARN] Unable to close REST client session. May already be closed: %s", errorRestClient.Error())
```